Calculator/Calculator/Model/ExpressionParser.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ExpressionParser.swift
+//  Calculator
+//
+//  Created by Erick on 2023/06/01.
+//
+
+enum ExpressionParser {
+    static func parse(from input: String) -> Formula {
+        var operands = CalculatorItemQueue<Double>()
+        var operators = CalculatorItemQueue<Operator>()
+        let components = componentsByOperators(from: input)
+        
+        components
+            .compactMap { Double($0) }
+            .forEach { operands.enqueue($0) }
+        
+        components
+            .filter { $0.count == 1 }
+            .compactMap { Operator(rawValue: Character($0))}
+            .forEach { operators.enqueue($0)}
```

Comment:
아! 지금보니 띄어쓰기 오타가 있네요 ㅎㅎ
```suggestion
            .compactMap { Operator(rawValue: Character($0)) }
            .forEach { operators.enqueue($0) }
```

Reply:
띄어쓰기 수정했습니다!